kernel/packages/unity-interface/UnityParcelScene.ts

```diff
@@ -22,15 +22,45 @@ export class UnityParcelScene extends UnityScene<LoadableParcelScene> {
     gridToWorld(this.data.data.basePosition.x, this.data.data.basePosition.y, aux)
     worker.setPosition(aux)
 
-    this.worker.getAPIInstance(DevTools)
-      .then(devTools => devTools.logger = this.logger)
+    this.worker
+      .getAPIInstance(DevTools)
+      .then((devTools) => (devTools.logger = this.logger))
       .catch((e) => this.logger.error('Error initializing system DevTools', e))
 
-    this.worker.getAPIInstance(ParcelIdentity)
-      .then(parcelIdentity => {
+    this.worker
+      .getAPIInstance(ParcelIdentity)
+      .then((parcelIdentity) => {
         parcelIdentity.land = this.data.data.land!
         parcelIdentity.cid = worker.getSceneId()
       })
       .catch((e) => this.logger.error('Error initializing system ParcelIdentity', e))
   }
 }
+
+export class UnityPortableExperienceScene extends UnityScene<LoadablePortableExperienceScene> {
+  constructor(public data: EnvironmentData<LoadablePortableExperienceScene>) {
+    super(data)
+    let loggerPrefix = data.data.basePosition.x + ',' + data.data.basePosition.y + ': '
```

Comment:
Does it make sense to have this as a prefix? Aren't all base positions for PEs 0,0? Wouldn't it be better if the prefix was the PE's id?

Reply:
done